clio-client/src/main/scala/org/broadinstitute/clio/client/metadata/CramDeliverer.scala

```diff
@@ -37,7 +38,8 @@ class CramDeliverer extends MetadataMover[CramMetadata] {
       writeMd5Op.toIterable
     ).flatten
 
-    if (src.regulatoryDesignation.exists(_.equals(RegulatoryDesignation.ResearchOnly))) {
+    if (deliverMetrics &&
+        src.regulatoryDesignation.exists(_.equals(RegulatoryDesignation.ResearchOnly))) {
```

Comment:
Now that this feature is opt-in only, does the code still need to check for `ResearchOnly`? We should find out from ops and/or Janki how they expect this feature to be used.

Reply:
I agree. As of now, it acts as an extra safeguard